keyboards/sleepy_craft_studios/sleepy_keeb_split/keyboard.json

```
@@ -0,0 +1,100 @@
+{
+    "manufacturer": "Sleepy Craft Studios",
+    "keyboard_name": "sleepy_keeb_split",
+    "maintainer": "Sleepy Craft Studios",
+    "development_board": "promicro",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "matrix_pins": {
+        "cols": ["B5", "B6", "B2", "B3", "B1", "F7"],
+        "rows": ["C6", "D7", "E6", "B4"]
+    },
+    "rgblight": {
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "static_gradient": true,
+            "twinkle": true
+        },
+        "led_count": 9,
+        "split": true
```

I suspect that this isn't accurate.  

9 on each side? If so:
```suggestion
        "led_count": 18,
        "split_count": [9, 9],
```

---

Ah good catch, I thought the count was for each side. Fixed!